File: public/codesamples/posts/graphes_grids/code2/Connection.java
package posts.graphes_grids.code2;

import posts.graphes_grids.code1.Node;

public class Connection {
  private Node from;
  private Node to;

  public Connection(Node from, Node to) {
    this.from = from;
    this.to = to;
  }

  public Node getFrom() {
    return from;
  }

  public Node getTo() {
    return to;
  }
}

File: public/codesamples/posts/graphes_grids/code2/Connection.cs
using Posts.GraphesGrids.Code1;
namespace Posts.GraphesGrids.Code2
{
    public class Connection
    {
        private Node _from;
        public Node from
        {
            get { return from; }
        }
        public Node _to;
        public Node to
        {
            get { return to; }
        }
        public Connection(Node from, Node to)
        {
            _from = from;
            _to = to;
        }
    }
}
